6-datascience/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:6.10
```

Comment:
Oh nice, was wondering about that.  Does docker automatically use the latest `.x` build in the repo?

https://github.com/nodejs/docker-node
![image](https://cloud.githubusercontent.com/assets/5067638/24260253/8db28276-0fb0-11e7-9202-919df79d5624.png)


So, if they add `6.11`, will it automatically grab that?